Minecraft/src/main/java/com/microsoft/Malmo/MissionHandlers/MazeDecoratorImplementation.java

```diff
@@ -162,7 +160,7 @@ private void initBlocksAndHeights()
             {
                 List<String> items = this.mazeParams.getWaypoints().getWaypointItem();
                 String item = items.get(this.blockrand.nextInt(items.size()));
-                this.waypointItem = MinecraftTypeHelper.ParseItemType(item);
+                this.waypointItem = MinecraftTypeHelper.ParseItemType(item, true);// ItemStackTODO...
```

Comment:
Good spot. MazeDecorator waypoints may be a loose end still, but will sort that out separately.